Formula/cfonts.rb

```diff
@@ -4,6 +4,12 @@ class Cfonts < Formula
   url "https://github.com/dominikwilkowski/cfonts/archive/refs/tags/v1.1.0rust.tar.gz"
   sha256 "45c40dfc867234efc5c5a2df687ccfc40a6702fa5a82f2380b555f9e755508e6"
   license "GPL-3.0-or-later"
+  head "https://github.com/dominikwilkowski/cfonts.git", branch: "released"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)rust$/i)
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)[._-]?rust$/i)
```

Some of the older versions use a hyphen between the version and `rust` (e.g., `v1.0.0-rust`) instead of the current `v1.1.0rust` format without a hyphen. We should match either format so we don't miss a version if there's variation in the future.